src/main/webapp/WEB-INF/tags/instructor/copyModal.tag

```diff
@@ -4,7 +4,7 @@
      aria-labelledby="fsCopyModal" aria-hidden="true">
     <div class="modal-dialog">
         <div class="modal-content">
-            <form method="post" name="form_copy_list" role="form"
+            <form method="post" id="form_copy_list" role="form"
```

Comment:
Now that it's an ID, we would probably want it to follow the naming convention we have for CSS?
(For classes, hyphens are used. For ids, it seems to be camelCase? We seem to be inconsistent between using camelCase and snake_case.)
(Also a more descriptive name regarding its role as the session copy modal, please.)

An alternative would be simply using ``[name=___]`` in that JQuery selector, but names on form tags seems to be legacy nowadays compared to ids?